### DISCUSSION TITLE
Filter tests by filename in watch mode

### DISCUSSION THREAD

**Discussion by netcoding87** [vitest-dev/vitest#1796] [Q&A] [answered]
Hi there,

first of all many thanks for this great development 👍 I am missing the option to filter test by filename in watch mode 😞  I am coming from jest where I really often used this filter (jest-watch-typeahead/filename) and it is very helpful for me. Have I overseen sth or is it possible to add this filter option somehow?

vitest
<img width="354" alt="image" src="https://user-images.githubusercontent.com/54022880/183005909-0fde9d48-f62a-44be-b25e-57b247248a47.png">

jest
<img width="369" alt="image" src="https://user-images.githubusercontent.com/54022880/183005947-86a02414-9743-4a2e-9f00-8d1434536b3c.png">

Many thanks,
Nick

**Comment by sheremet-va** [comment, marked as answer]
There is no such feature yet. You can open an issue for enhancement.
